### PR TITLE
Use nop binary instead of /bin/true

### PR DESCRIPTION
**Pull request by chmouel**

### CODE REVIEW

File: openshift/ci-operator/tekton-images/nop/Dockerfile [hunk 1]
```diff
@@ -9,4 +9,4 @@ FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
 RUN for i in /usr/bin/* /usr/sbin/* /bin/* /sbin/*;do \
     [[ $i == *bin/true || $i == *bin/rm || $i == *bin/coreutils ]] && continue;rm -vf $i;done
 
-ENTRYPOINT ["/bin/true"]
\ No newline at end of file
+ENTRYPOINT ["/ko-apps/nop"]
```
Comment (piyush-garg) [inline openshift/ci-operator/tekton-images/nop/Dockerfile:12]: We need to add nop cmd also

```
ADD nop /ko-app/nop
ENTRYPOINT ["/ko-app/nop"]
```

Comment (chmouel) [reply openshift/ci-operator/tekton-images/nop/Dockerfile:13]: yes thanks! i send it too quickly,

File: openshift/ci-operator/tekton-images/nop/Dockerfile [hunk 2]
```diff
@@ -7,6 +7,7 @@ FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
 # still be based on a UBI, it's not pretty but let's hope we can remove this when
 # we get https://github.com/tektoncd/pipeline/issues/1347
 RUN for i in /usr/bin/* /usr/sbin/* /bin/* /sbin/*;do \
-    [[ $i == *bin/true || $i == *bin/rm || $i == *bin/coreutils ]] && continue;rm -vf $i;done
+    [[ $i == *bin/rm || $i == *bin/coreutils ]] && continue;rm -vf $i;done
 
-ENTRYPOINT ["/bin/true"]
\ No newline at end of file
+ADD nop /ko-app/nop
+ENTRYPOINT ["/ko-apps/nop"]
```
Comment (piyush-garg) [inline openshift/ci-operator/tekton-images/nop/Dockerfile:13]: nit: ko-apps to ko-app

Comment (chmouel) [reply openshift/ci-operator/tekton-images/nop/Dockerfile]: 🤦

**Pull request merged**